client/schema/LBSettings.schema.json

```diff
@@ -76,7 +76,11 @@
             "type": "array",
             "items": { "type": "string", "pattern": "^(\\d{3}|all)$" }
           },
-          "CacheTime": { "$ref": "#/definitions/nonEmptyString" }
+          "CacheTime": { "$ref": "#/definitions/nonEmptyString" },
+          "RestrictTo": {
```

Comment:
When you have shared patterns, they can be added as a `type` in the definitions section of the schema document. 

`"pattern": "#/definitions/typeNameForRegex"`